notification-stream/package.json

```diff
@@ -10,6 +10,8 @@
     "author": "Viafoura",
```

Comment:
can we add "engines" so we are clear what version(s) of Node this is compatible with?